client/src/index.css

```
@@ -342,6 +342,80 @@
+  --chart-4: 45 100% 40%;
+  --chart-5: 300 50% 35%;
+
+  --radius-sm: 0px;
+  --radius-md: 0px;
+  --radius-lg: 0px;
+  --radius-xl: 0px;
+  --radius-2xl: 0px;
+  --radius-3xl: 0px;
+  --radius: 0px;
+
+  --font-sans: Tahoma, "Segoe UI", "MS Sans Serif", Geneva, Verdana, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Tahoma" to be "tahoma" (value-keyword-case)

(value-keyword-case)